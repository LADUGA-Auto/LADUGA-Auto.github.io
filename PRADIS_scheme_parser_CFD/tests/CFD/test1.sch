<Qucs Schematic 4.4.0>
<Properties>
  <View=0,0,800,800,1,0,0>
  <Grid=10,10,1>
  <DataSet=test1.dat>
  <DataDisplay=test1.dpl>
  <OpenDisplay=1>
  <showFrame=0>
  <FrameText0=\x041D\x0430\x0437\x0432\x0430\x043D\x0438\x0435>
  <FrameText1=\x0427\x0435\x0440\x0442\x0438\x043B:>
  <FrameText2=\x0414\x0430\x0442\x0430:>
  <FrameText3=\x0412\x0435\x0440\x0441\x0438\x044F:>
</Properties>
<Symbol>
</Symbol>
<Components>
  <CoSimulation.Port Port1 1 110 120 -41 29 0 0 "firstP" 0 "Fluid" 0>
  <CoSimulation.Port Port2 1 200 120 -41 29 0 0 "secondP" 0 "ThermalFluid" 0>
  <CoSimulation.Port Port3 1 290 120 -41 29 0 0 "thirdP" 0 "Thermal" 0>
  <CoSimulation.Port Port4 1 380 120 -41 29 0 0 "fourthP" 0 "XYZ" 0>
  <CoSimulation.FluidDomain FluidDomain1 1 150 220 -71 29 0 0 "water" 0 "10" 0 "1" 0 "20" 0>
  <CoSimulation.SolidDomain SolidDomain1 1 330 220 -71 29 0 0 "stone" 0 "2" 0 "25" 0 "3" 0>
  <CoSimulation.Task Task1 1 240 420 -41 29 0 0 "example" 0 "text" 0>
  <CoSimulation.Solver Solver1 1 240 320 -51 29 0 0 "5" 0 "2" 0 "1" 0 "" 0>
</Components>
<Wires>
</Wires>
<Diagrams>
</Diagrams>
<Paintings>
</Paintings>
