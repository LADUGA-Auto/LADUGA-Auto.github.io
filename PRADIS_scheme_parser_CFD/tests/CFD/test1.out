[ Module ]      |      [ Name ]       |      [ Active ]       |      [ Properties ]
[ CoSimulation.Port ] | [ Port1 ] | [ true ]  |  [ Name : firstP : string ]    [ Interface : Fluid : string ]    

[ CoSimulation.Port ] | [ Port2 ] | [ true ]  |  [ Name : secondP : string ]    [ Interface : ThermalFluid : string ]    

[ CoSimulation.Port ] | [ Port3 ] | [ true ]  |  [ Name : thirdP : string ]    [ Interface : Thermal : string ]    

[ CoSimulation.Port ] | [ Port4 ] | [ true ]  |  [ Name : fourthP : string ]    [ Interface : XYZ : string ]    

[ CoSimulation.FluidDomain ] | [ FluidDomain1 ] | [ true ]  |  [ Fluid : water : string ]    [ Pressure : 10 : integer ]    [ Velocity : 1 : integer ]    [ Temperature : 20 : integer ]    

[ CoSimulation.SolidDomain ] | [ SolidDomain1 ] | [ true ]  |  [ Solid : stone : string ]    [ Velocity : 2 : integer ]    [ Temperature : 25 : integer ]    [ RotationalVelocity : 3 : integer ]    

[ CoSimulation.Task ] | [ Task1 ] | [ true ]  |  [ File : example : string ]    [ Description : text : string ]    

[ CoSimulation.Solver ] | [ Solver1 ] | [ true ]  |  [ EndTime : 5 : integer ]    [ MaximalIterationCount : 2 : integer ]    [ Tolerance : 1 : integer ]    [ Flag.GetJacobian :  : string ]    

